deploy/test-build/ssl/f7t_internal.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
On a second thought, these may no longer be needed, if we agreed to go without ssl for the `test-build` cluster. To keep in mind. @fcruzcscs PTAL at the SSL change introduced in [test-build](https://github.com/eth-cscs/firecrest/pull/76/files#diff-df8189504f0811599327a3161d585936fabc2689aef146c9745bab847e972074)

Reply:
For `test-build` it could be surelly removed, we'll keep it until next release.

Reply:
in the future we should move SSL and other security features to a service mesh, but that is future work